### ISSUE TITLE
LandmarkNamePartTable Issues

### ISSUE THREAD

**Issue by geofinne** [NENA911/NG911GISDataModel#38]
**Description**

1. For the LandmarkNamePartTable, the “NGUID” field is **nullable** and **non-required** in the geodatabase script but the standard has the field as required.

2. In both the geodatabase and PostgreSQL scripts for the LandmarkNamePartTable, the “LMNP_Order” field domain “LandmarkNamePartOrder” has a width of 2 and the standard calls for the field width to be 1.

**Comment by tomneer** [comment]
_For the LandmarkNamePartTable, the “NGUID” field is nullable and non-required in the geodatabase script but the standard has the field as required._

Confirmed in [schema_fgdb_v2.py](https://github.com/NENA911/NG911GISDataModel/blob/986686da9ae68cdcadf9766cd5943f93969fbbf9/esri_geodatabase/NENA_NG911_Scripts/schema/schema_fgdb_v2.py#L2740).

_In both the geodatabase and PostgreSQL scripts for the LandmarkNamePartTable, the “LMNP_Order” field domain “LandmarkNamePartOrder” has a width of 2 and the standard calls for the field width to be 1._

The working group should probably discuss this issue. Generally, field widths only apply to character fields (e.g., VARCHAR, TEXT, etc.) and not numeric fields (e.g., SHORT, INTEGER). We used what was defined in v1 which includes a domain constraint of values between 1 and 99. Below is the field description from NENA-STA-006.2-2022, which only sets a domain of whole numbers but not an actual range. My assumption is that v1 may have determined the range. @jasonhorning - We may want to add this to our items to discuss.

```
5.49 Landmark Name Part Order
     Description: The order in which to concatenate Landmark Name Parts where 1 
     is the first (or leftmost) Landmark Name Part, 2 is the second Landmark Name 
     Part, 3 is the third Landmark Name Part, etc.
     Domain: Whole numbers starting at 1
     Example: 1; 2; 3
```
